src/components/ActionControls/ActionControls.scss

```diff
@@ -0,0 +1,42 @@
+@import '../../variables';
+
+.ba {
+    .ba-action-controls {
+        color: $fours;
+        display: flex;
+        min-width: 0;
+        padding: 10px 5px;
+        white-space: normal;
+        width: 100%;
+        z-index: 130;
```

Comment:
Is `z-index` necessary here? It's usually best to avoid setting them, if possible.